acoustid/future/fpindex/client.py

```
@@ -79,6 +79,40 @@ class FingerprintIndexClientError(Exception):
+                    id=id,
+                    hashes=hashes,
+                )
+            )
+        )
+
+    def delete(self, id: int) -> None:
+        self.changes.append(
+            Change(
+                delete=Delete(
+                    id=id,
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)